### ISSUE TITLE
`skip` in a sequence seems absolute rather than an upper limit

### ISSUE THREAD

**Issue by CollierCZ** [jdkato/txtlint#498]
When using `skip` in a `sequence` rule, it seems to be requiring that exact number of words in the middle, rather than being up to that many words, as I'd expect from the [docs](https://vale.sh/docs/topics/styles/#sequence).

Example rule:

```
extends: sequence
message: "Use data and metadata as plural nouns."
level: suggestion
ignorecase: true
tokens:
  - tag: NN
    pattern: '(?:data|metadata)'
    skip: 4
  - tag: VBZ
 ```

This flags the following: `Metadata for this one purpose is valid.`

But if I remove a word, it isn't flagged: `Metadata for this purpose is valid.`

Is this expected behavior? Is there some other way to achieve the goal of not knowing how many words might be in between the subject and verb?

I also tried `skip` as its own item in the list (`- skip`), but that just flagged anything with `data` or `metadata` and two words after it.

I'm on Vale 2.20.2 on Linux installed with Homebrew

**Comment by theletterf** [comment]
Can confirm the issue.

**Comment by theletterf** [comment]
For context, this is the rule that would benefit from `skip` in our style guide:

https://github.com/splunk/vale-splunk-style-guide/blob/main/styles/Splunk/DataSingular.yml

**Comment by jdkato** [comment]
This should be fixed in the next release.

(Although the `skip` key should be on `tag: VBZ` entry.)

**Comment by CollierCZ** [comment]
Thanks so much!

**Comment by theletterf** [comment]
Thanks, @jdkato !
